platform/darwin/src/MGLBackgroundStyleLayer.h

```diff
@@ -12,6 +12,7 @@ NS_ASSUME_NONNULL_BEGIN
  `style` and obtain the background layer using the `-[MGLStyle layerWithIdentifier:]` 
  method and passing `background` for the identifier. 
  */
+MGL_EXPORT
```

Comment:
This change will get blown away as soon as anyone invokes `make style-code`. Make this change in platform/darwin/src/MGLStyleLayer.h.ejs and run `make style-code` to propagate the change to all the affected headers.